R/estimateDiversity.R

```diff
@@ -183,7 +183,9 @@
 #' "Faith",  "LogModSkewness")
```

Comment:
does this function estimateDiversity now have a deprecation message?

Reply:
yes